addOns/automation/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 - Support for exclude regexes to active scan config job.
 - Always run option for all jobs.
+- Support for data driven nodes in the yaml (not in the UI).
```

Comment:
"...(not yet in the UI)"?

So that it doesn't sound like it's totally never going to happen :grinning: